controllers/idm_controller_test.go

```diff
@@ -94,7 +94,7 @@ var _ = Describe("LOCAL:IDMReconciller", func() {
 				},
 			}
 			Expect(idm).ShouldNot(BeNil())
-			pod := manifests.MainPodForIDM(idm, "localhost", "ephemeral")
+			pod := manifests.MainPodForIDM(idm, "localhost", "quay.io/freeipa/freeipa-openshift-container:freeipa-server", "ephemeral")
```

Comment:
If Pod object manifest is not used, clean-up the code, or mark the block to be clean-up with a TODO.